### PR TITLE
Adding carousel and member pages

### PR DESCRIPTION
**Pull request by sirjessthebrave**

### CODE REVIEW

**Comment by sirjessthebrave** [comment]
@allypalanzi what do you think about the carousel before we merge it?

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .DS_Store
-_site
```
Comment (allypalanzi) [inline .gitignore:2]: this should stay git ignored, we dont need the compiled files in the repo

**Comment by allypalanzi** [comment]
Can you pull the added groups out of this and merge that in separately? I think instead of hte carousel I'll work on a better design for this section.

**Comment by sirjessthebrave** [comment]
yah, i will just 'turn off' the carousel part :)

File: _includes/head.html [hunk 2]
```diff
@@ -4,7 +4,35 @@
...
+
+  <link rel="stylesheet" href="{{ 'css/main.css' | prepend: site.baseurl }}">
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
   <link rel="icon" type="image/png" href="assets/fav.png">
+
+  
+  <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
+  <script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+
+  <!-- slick jquery carousel  -->
```
Comment (allypalanzi) [inline _includes/head.html]: instead of commenting this out can we just remove it? same with the jquery above if we're not using js

**Comment by sirjessthebrave** [comment]
OK updated turning carousel off!

File: _includes/members.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<ul class="m-members">
+<ul class="m-members carousel-wrapper">
```
Comment (allypalanzi) [inline _includes/members.html]: pls remove carousel-wrapper since we don't have this anymore

File: icons/pyladies-geek.ai [hunk 4]
```diff
@@ -0,0 +1,460 @@
+%PDF-1.5%����
```
Comment (allypalanzi) [inline icons/pyladies-geek.ai]: we prob dont want an .ai file in here

File: icons/pyladies-logo.ai [hunk 5]
```diff
@@ -0,0 +1,359 @@
+%PDF-1.5%����
```
Comment (allypalanzi) [inline icons/pyladies-logo.ai]: another ai file

**Comment by sirjessthebrave** [comment]
Ok that should be updated I think

**Comment by allypalanzi** [comment]
@sirjessthebrave can you re-add site back into the gitignore?

**Comment by sirjessthebrave** [comment]
got it

**Comment by allypalanzi** [comment]
ahh so all of the `_site/` files are still being tracked 😬 can you delete the folder so it stops tracking them? (i cant view the diff with them in this PR bc it freezes up github 😢)

File: .gitignore [hunk 6]
```diff
@@ -2,3 +2,5 @@
 
 .sass-cache
 .jekyll-metadata
+
+-_site
```
Comment (allypalanzi) [inline .gitignore]: oh this shouldn't have a `-`

**Comment by sirjessthebrave** [comment]
Trying again :P  I am doing this all way to fast between meetings - never good for dev!

File: readme.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# DCFem Tech Site (http://allypalanzi.github.io/dcft-website/index.html)
+# DCFem Tech Site (https://sirjessthebrave.github.io/dcft-website/)
```
Comment (allypalanzi) [inline readme.md:1]: we'll wanna remove this before merging into mine

Comment (allypalanzi) [inline readme.md:1]: as soon as this is fixed and the ai files are removed, you can merge this in! 🎉

**Pull request merged**